doc/news/2022-05-01-markor-v2.9.md

```diff
@@ -41,12 +41,159 @@ This search now also checks contents of files encrypted by Markor.
+* [Update translations (PR #1654)](https://github.com/gsantner/markor/commit/40f39e99)
+* [Replace discussion reference from Matrix to GitHub discussions, by @gsantner](https://github.com/gsantner/markor/commit/3514491d)
+* [Filebrowser: Fix copy/move getCurrentFolder() NullPointer, by @gsantner](https://github.com/gsantner/markor/commit/7a3ffd04)
+* [Fix ArrayIndexOutOfBounds at ShortcutUtils, by @gsantner](https://github.com/gsantner/markor/commit/e746478c)
+* [Filemanager: disable search button at root directory, closes #1652, by @gsantner](https://github.com/gsantner/markor/commit/47de5795)
+* [Base functionality to handle Toolbar (title) clicks & long-clicks, by @gsantner](https://github.com/gsantner/markor/commit/6ac968c5)
+* [Update translations (PR #1649)](https://github.com/gsantner/markor/commit/016f2289)
+* [Simplify TextAction handling code, by @harshad1 (PR #1626)](https://github.com/gsantner/markor/commit/f2aa4b59)
+* [markdown: Fix rendered list indent with depth 3, closes #1643, by @fgtham (PR #1651)](https://github.com/gsantner/markor/commit/302b022f)
+* [Don't run auto-format immediately on document load, by @gsantner](https://github.com/gsantner/markor/commit/6014327f)
+* [Update translations (PR #1633)](https://github.com/gsantner/markor/commit/dedb044f)
+* [Fix version of AppIntro dependency, by @gsantner, closes #1632](https://github.com/gsantner/markor/commit/285551f2)
+* [Feature: Snippets, by @harshad1, closes #437 (PR #1624)](https://github.com/gsantner/markor/commit/3f7243b0)
+* [Update translations (PR #1627)](https://github.com/gsantner/markor/commit/6c411b7b)
+* [Automatically set HTML lang attribute, by @fgtham (PR #1623)](https://github.com/gsantner/markor/commit/cd3d0fec)
+* [Update translations (PR #1619)](https://github.com/gsantner/markor/commit/2c8c682b)
+* [TextActions improvements for spacing and save, by @harshad1 (PR #1622)](https://github.com/gsantner/markor/commit/86a55620)
+* [textactions: allow setting lower spacing, by @harshad1 (PR #1621)](https://github.com/gsantner/markor/commit/c7fdadde)
+* [wording improvements](https://github.com/gsantner/markor/commit/b46291f8)
+* [Markdown: Fix wrong indentation of code blocks in checklists, by @fgtham, closes #1600 (PR #1620)](https://github.com/gsantner/markor/commit/98e9e83e)
+* [markdown: YAML front-matter improvements, by @gsantner (PR #1597)](https://github.com/gsantner/markor/commit/8fb99feb)
+* [Update translations (PR #1616)](https://github.com/gsantner/markor/commit/89d8fec0)
+* [rename string sort 'Sort by' -> sort_by, by @gsantner](https://github.com/gsantner/markor/commit/d69e1abf)
+* [markdown: Support and display YAML front-matter block contents, by @fgtham @gsantner (PR #1597)](https://github.com/gsantner/markor/commit/dac68bf4)
+* [Reformat code, optimize imports](https://github.com/gsantner/markor/commit/d9c78be2)
```

Comment:
Minor and internal

Reply:
You can ignore all those lines. It is pure git history :smile: .

What matters is the content of the CHANGELOG.md file (there didn't add anything yet). And of course, if you want to say something about some new feature - describing it nicely with optional graphics - example -> <https://gsantner.net/blog/2021/07/29/markor-v2.8-todotxt-dialog-contexts-projects-multi-selection.html>